core/src/test/scala/unit/kafka/cluster/PartitionLockTest.scala

```diff
@@ -309,8 +309,11 @@ class PartitionLockTest extends Logging {
     }
     when(offsetCheckpoints.fetch(ArgumentMatchers.anyString, ArgumentMatchers.eq(topicPartition)))
       .thenReturn(None)
-    when(alterIsrManager.submit(ArgumentMatchers.any[AlterIsrItem]))
-      .thenReturn(true)
+    when(alterIsrManager.submit(
+      ArgumentMatchers.eq(topicPartition),
+      ArgumentMatchers.any[LeaderAndIsr],
+      ArgumentMatchers.anyInt()))
+      .thenReturn(new CompletableFuture[LeaderAndIsr]())
```

Comment:
nit: These two lines read a bit weird. I would move the closing parenthesis of `when` and `submit` to the next line, before `.thenReturn` to make it a bit more readable.

```
when(alterIsrManager.submit(
  ArgumentMatchers.eq(topicPartition),
  ArgumentMatchers.any[LeaderAndIsr],
  ArgumentMatchers.anyInt()
)).thenReturn(new CompletableFuture[LeaderAndIsr]())
```